models/classes/Tool/Validation/Lti1p3Validator.php

```diff
@@ -90,6 +102,17 @@ public function validateRole(LtiMessagePayloadInterface $ltiMessagePayload): voi
 
     private function getRegistrationRepository(): RegistrationRepositoryInterface
     {
-        return $this->getServiceLocator()->getContainer()->get(RegistrationRepositoryInterface::class);
+        return $this->registrationRepository;
+    }
+
+    /**
+     * @return ToolLaunchValidator
+     */
+    private function getToolLaunchValidator(): AbstractLaunchValidator
```

Comment:
its private function not public getter so maybe for clean code we can name it as createLaunchValidator() or something similrar?